ladder.py

```diff
@@ -116,12 +115,12 @@ def eval_batch_norm():
             z = batch_normalization(z_pre, mean, var)
             # Instead of the above statement, the use of the following 2 statements containing a typo
             # consistently produces a 0.2% higher accuracy for unclear reasons.
-            # m_l, v_l = tf.nn.moments(z_pre_l, axes=[0])
-            # z = join(batch_normalization(z_pre_l, m_l, mean, var), batch_normalization(z_pre_u, mean, var))
+            #m_l, v_l = tf.nn.moments(z_pre_l, axes=[0])
```

Comment:
Nice work and thanks for the pull request. I believe it is good coding style to have a space after the #. Can you submit a pull request with the change?